modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,151 @@
+
+* After the upgrade to Server 10.3 has finished successfully, keep the maintenance mode activated and install/enable `files_external_s3` (either manually or via the Market app) as the app is not bundled with ownCloud Server anymore.
+* After enabling `files_external_s3` (`occ app:enable files_external_s3`) it is necessary to run `occ upgrade` again to conduct config migrations. This migration step should run quickly.
+* If users were allowed to configure personal mount points before the upgrade, switch from maintenance mode into single user mode (`occ maintenance:singleuser --on`) and enable the option again by ticking the respective checkbox below "Allow users to mount external storage" (in Admin settings => Storage).
+* Existing storage mount points will stay and do not have to be touched.
+* Make sure that everything works and disable maintenance/single user mode to put the installation back into normal operations (`occ maintance:mode --off` / `occ maintenance:singleuser --off`).
+
+=== New HTTP APIs
+
+ownCloud Server is being prepared for https://owncloud.org/news/owncloud-phoenix-rebirth-of-the-owncloud-user-interface/[Phoenix], the upcoming web frontend for ownCloud. As Phoenix is clearly separated from the backend and communicates only via HTTP APIs it is necessary to complete the API coverage.
+
+The following new HTTP APIs have been added with Server 10.3:
+
+* WebDAV Trash bin API (https://doc.owncloud.com/server/developer_manual/webdav_api/trashbin.html[Developer documentation])
+* OCS API for public link share email notifications ([Developer documentation])
+* WebDAV endpoint for public links ([Developer documentation])
+https://github.com/owncloud/docs/issues/1871
+
+All new endpoints are currently in tech preview state and are mainly used for Phoenix development. For this reason they are disabled by default and have to be explicitly enabled using the new config.php option `'dav.enable.tech_preview' => true,`.
+
+=> **Link to dev docs**
+
+=== Other notable changes
+
+* The `previews_path` config option has been added to allow customization of the thumbnail storage path (by default those reside in the user storage). https://github.com/owncloud/core/pull/35131[#35131]
+* An Activity entry will now be shown when a share receiver unshares a share https://github.com/owncloud/core/pull/35193[#35193]
+* The WebUI experience on mobile devices has been improved https://github.com/owncloud/core/pull/35919[#35919] https://github.com/owncloud/core/pull/35813[#35813] https://github.com/owncloud/core/pull/35347[#35347] https://github.com/owncloud/core/pull/34803[#34803] 
+* Two new config.php options (`proxy` and `proxyuserpwd`) have been added to enable federation when an instance needs to go through an authenticated proxy to reach a federated instance. See `config.sample.php` and the **LINK** [ownCloud Documentation] for more information. https://github.com/owncloud/core/pull/35868[#35868]
```

Comment:
They are not new, but federation ignored them in the past, They were already working for the updater

Reply:
thx, changed